### DISCUSSION TITLE
What does 'unstable' mean?

### DISCUSSION THREAD

**Discussion by hayer** [O-X-L/ansible-opnsense#85] [Q&A] [answered]
I see in the `README.md` that `ansibleguy.opnsense.unbound_acl`, and others, are marked as "unstable".
What does this mean? Are there any specific details on the different items and why they are marked as unstable?
I would love to help make them stable.

**Comment by superstes** [comment, marked as answer]
Greetings.

These are the development states:

`not implemented => development => testing => unstable (practical testing) => stable`

Unstable = Module is working but may still have issues for edge cases.

The state is basically set to stable if no issues are reported over some months.
